seatunnel-connectors-v2/connector-redis/src/main/java/org/apache/seatunnel/connectors/seatunnel/redis/config/RedisConfig.java

```diff
@@ -60,7 +61,19 @@ public enum HashKeyParseMode {
         Options.key("keys")
             .stringType()
             .noDefaultValue()
-            .withDescription("keys pattern, redis source connector support fuzzy key matching, user needs to ensure that the matched keys are the same type");
+            .withDescription("keys pattern, redis source connector support fuzzy key matching, if keys_type_check is false, user needs to ensure that the matched keys are the same type");
+
+    public static final Option<Boolean> KEY_TYPE_CHECK =
```

Comment:
Please update the document and add change log about this pr.

Reply:
@EricJoy2048 Update completed